website/content/docs/upgrading/upgrade-to-1.9.x.mdx

```diff
@@ -95,3 +95,13 @@ respects the order of suites given in `tls_cipher_suites`.
 
 See [this blog post](https://go.dev/blog/tls-cipher-suites) for more information.
 
+## Known Issues
+- Existing Vault installations that use the [Identity Token
+backend](/api-docs/secret/identity/tokens) and have [named
+keys](/api-docs/secret/identity/tokens#create-a-named-key) generated will
+encounter a panic when any of those existing keys are rotated once passed its
+`rotation_period`. This issue affects Vault 1.9.0, and fixed in Vault 1.9.1. The
+workarounds for this issue are to either 1) delete any existing keys before the
+update, or 2) increase the rotation period to allow ample time for updating past
+1.9.0 before a rotation can occur. Workarounds are not necessary if an update to
```

Comment:
```suggestion
workarounds for this issue are to either:

1. Delete any existing keys before the update, or
1. Increase the rotation period to allow ample time for updating past 1.9.0 before a rotation can occur.

Workarounds are not necessary if an update
```

Reply:
👍

Reply:
Updated this part since it was not entirely accurate after further discovery (bumping the rotation period won't cut it).